src/robocoup/jobs.js

```diff
@@ -7,15 +8,21 @@ import {query} from '../lib/db';
 const scheduler = new Scheduler();
 export default scheduler;
 
-scheduler.add('00 00 09 * * 1', function() {
+scheduler.add('0 12 * * 1-5', function() {
```

Comment:
This doesn't error, but is it valid for this npm module? All the examples in the docs show 6 values, not 5.

https://github.com/ncb000gt/node-cron#usage-basic-cron-usage